packages/flutter/lib/src/widgets/sliver_fill.dart

```diff
@@ -41,13 +43,26 @@ class SliverFillViewport extends StatelessWidget {
   /// the viewport in the main axis.
   final double viewportFraction;
 
+  /// Whether to add padding to both ends of the list.
+  ///
+  /// If this is set to true and [viewportFraction] < 1.0, padding will be added
+  /// such that the first and last child slivers will be in the center of
+  /// the viewport when scrolled all the way to the start or end, respectively.
+  /// You may want to set this to false if this [SliverFillViewport] is not the only
+  /// widget along this main axis, such as in a [CustomScrollView] with multiple
+  /// children.
+  ///
+  /// This option cannot be [null]. If [viewportFraction] >= 1.0, no padding will be
+  /// added regardless, so this option does nothing. Defaults to [true].
```

Comment:
The second sentence is a little confusing, does this say what you mean?
```dart
/// This option cannot be [null]. If [viewportFraction] >= 1.0, this option has no effect.
/// Defaults to [true].
```

Reply:
I like that a bit better 😄